src/averageAges.js

```diff
@@ -54,8 +67,26 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+  const kids = (onlyWithSon
+    ? [...people].filter(person => person.sex === 'm')
```

Comment:
you can use `people.filter` without spread operator because `filter` method returns a new array.